content/blog/harbor-2.9.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Harbor v2.9 release"
+author:
+  name: "Orlin Vasilev"
+date: 2023-08-18T06:00:00+01:00
+draft: false
+showPageInfo: true
+---
+
+# Harbor 2.9
+
+##### Introducing [Harbor 2.9][r29]
+
+This release comes with several new features and improvements, including Harbor Security Hub and OCI v1.1.0-rc2, which provides enhanced support for multi-architecture images and image manifests. Another major addition is improved Garbage Collector. [The 2.9 release][r29] also includes an Customizable Message Banner , providing more comprehensive and detailed information about upcoming maintenance and admin activities. Also we have introduces some significant changes as the removal of Notary.
+
+## New features
+
+### Harbor Security Hub
+Admin users can now access valuable security insights, which including the number of scanned and unscanned artifacts, identification of dangerous artifacts and CVEs and advanced search capabilities for vulnerabilities using multiple combined conditions.
+
+* Add Security Hub UI by @AllForNothing in https://github.com/goharbor/harbor/pull/18942
+* Update table scan_report and extract cvss_v3_score from vendor attribute by @stonezdj in https://github.com/goharbor/harbor/pull/18854
+* Add vulnerability search API by @stonezdj in https://github.com/goharbor/harbor/pull/18924
+* Add security hub summary API by @stonezdj in https://github.com/goharbor/harbor/pull/18872
+* Create index in vulnerability_record table by @stonezdj in https://github.com/goharbor/harbor/pull/18949
+
+[Harbor Security Hub Documentation](https://goharbor.io/docs/2.9.0/administration/security-hub/)
+
+### GC Enhancements
+Improved visibility with detailed GC execution history and enable parallel deletion for faster GC triggers.
+
+* Add worker parameter for GC by @AllForNothing in https://github.com/goharbor/harbor/pull/18882
+* add more details in gc history by @wy65701436 in https://github.com/goharbor/harbor/pull/18779
+* add multiple deletion of GC by @wy65701436 in https://github.com/goharbor/harbor/pull/18855
+
+
+### [OCI Distribution Spec v1.1.0-rc2 Support](https://github.com/opencontainers/image-spec/releases/tag/v1.1.0-rc2)
+
+The latest Harbor release now supports the OCI Distribution Spec v1.1.0-rc2. This is a significant upgrade that allows users to store and distribute OCI and Docker images. With this feature, users can manage their images more effectively and efficiently.
```

Comment:
Should we also mention the customization we implemented for Notation (which is actually the Notary v2) and the Nydus integration? 

These integrations were developed based on the OCI Distribution Spec v1.1.0-rc2.